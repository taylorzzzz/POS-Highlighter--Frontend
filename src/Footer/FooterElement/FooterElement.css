.FooterElement {
    width: 50%;
    text-align: center;
    line-height: 1.5em;
}

.FooterElement.full { width: 100% }
.FooterElement.half { width: 50% }
.FooterElement.quarter { width: 25% }

.FooterElement:first-child {
    text-align: left;
}
.FooterElement:last-child {
    text-align: right;
}