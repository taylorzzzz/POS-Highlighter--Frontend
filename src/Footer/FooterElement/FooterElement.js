import React from 'react';

import './FooterElement.css';

function FooterElement(props) {

    const classes = props.classes
        ? [ ...props.classes, "FooterElement"]
        : [];


    return (
        <div className={classes.join(' ')}>
            {props.children}
        </div>
    )
}

export default FooterElement;