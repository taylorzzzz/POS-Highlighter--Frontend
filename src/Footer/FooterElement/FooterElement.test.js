import React from 'react';
import renderer from 'react-test-renderer';

import FooterElement from './FooterElement';

test('FooterElement renders properly', () => {
    const tree = renderer.create(<FooterElement />).toJSON();
    
    expect(tree).toMatchSnapshot();
})