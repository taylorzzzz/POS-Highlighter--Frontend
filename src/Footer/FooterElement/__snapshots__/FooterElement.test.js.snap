// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterElement renders properly 1`] = `
<div
  className=""
/>
`;
